frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteRepositoryDiscoverer.java

```diff
@@ -48,10 +49,12 @@ public void enqueue(Path macheteFilePath) {
     new Task.Backgroundable(project, getString("string.GitMachete.EnhancedGraphTable.automatic-discover.task-title")) {
       @Override
       public void run(ProgressIndicator indicator) {
+        LOG.info("Running automatic discover task");
```

Comment:
Hmmm maybe `LOG.debug` + then enable DEBUG for specific logger name (only for this class/package) in Rhino's `ide.configure(...)`?